.github/workflows/sessions-e2e.yml

```diff
@@ -1,7 +1,9 @@
 name: Firebase Sessions E2E Tests
 
 on:
-  workflow_dispatch: # allow triggering the workflow manually
+  schedule:
+    - cron: 4 */4 * * *  # every 4 hours at 04 minutes past the hour
```

Comment:
I think the other test also run at 4 minutes past the hour. It'd be good to randomize this value to distribute the runs.

Reply:
Done added 20 min